mmdet/models/detectors/panoptic_single_stage_segmentor.py

```diff
@@ -0,0 +1,107 @@
+import warnings
+
+from ..builder import DETECTORS, build_backbone, build_head, build_neck
+from .base import BaseDetector
+
+
+@DETECTORS.register_module()
+class SingleStageSegmentor(BaseDetector):
+    """Base class for single-stage segmentors.
+
+    Single-stage segmentors directly and densely predict mask on the
+    output features of the backbone+neck.
```

Comment:
The single-stage segmentor is not a Segmentor with only one semantic head.
Using the base head of DETR is ok for now.